plugin/src/main/java/io/jenkins/plugins/casc/TokenReloadAction.java

```diff
@@ -0,0 +1,90 @@
+package io.jenkins.plugins.casc;
+
+import com.google.common.base.Strings;
+import hudson.Extension;
+import hudson.model.UnprotectedRootAction;
+import hudson.util.Scrambler;
+import java.io.IOException;
+import java.util.logging.Logger;
+import javax.annotation.CheckForNull;
+import javax.servlet.http.HttpServletRequest;
+import org.apache.commons.httpclient.HttpStatus;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.interceptor.RequirePOST;
+
+/**
+ * If enabled by setting environment variable "CASC_ALLOW_LOCAL_RELOAD" to "true" it allows reloading
+ * configuration by issuing a POST request to "http://localhost:8080/reload-configuration-as-code/".
+ */
+@Extension
+public class TokenReloadAction implements UnprotectedRootAction {
+    public static final Logger LOGGER = Logger.getLogger(TokenReloadAction.class.getName());
+
+    public static final String URL_NAME = "/reload-configuration-as-code";
```

Comment:
thoughts on this endpoint name anyone?
maybe namespacing it is better?
```suggestion
    public static final String URL_NAME = "/configuration-as-code/reload-webhook";
```

Reply:
I like your suggestion, but changing the constant did not work. Seems like `UnprotectedRootAction` can't contribute to the existing `/configuration-as-code/`endpoint. Any suggestions how to do that?